fabricadesoftware-ifc/fabricaNode-BackEnd
# FIX: dataPublicacao on publicacao and dataNascimento on autor #12

### Changing type field

**Description:**
This issue change type date field to interger filed on autor and publicacao model

**Test:**

1. access the endpoint: "/admin"
2. click to add autor
3. check if dataNascimento field type is integer
4. click to add publicacao
5. check if dataPublicacao field type is integer
 

**Have you fulfilled the entire issue description?**
